kitsune/questions/cron.py

```diff
@@ -186,9 +186,11 @@ def report_employee_answers():
     num_answered = questions.filter(num_answers__gt=0).count()
 
     # Total number of questions answered by user in tracked_group
-    num_answered_by_tracked = Answer.objects.filter(
-        question__in=questions,
-        creator__in=tracked_users).values_list('question_id').distinct().count()
+    num_answered_by_tracked = {}
+    for user in tracked_users:
+        num_answered_by_tracked[user.username] = Answer.objects.filter(
+            question__in=questions,
+            creator=user).values_list('question_id').distinct().count()
```

Comment:
You can get this data in one query, instead of n: https://github.com/rlr/kitsune/blob/master/kitsune/users/api.py#L370-L374